### PR TITLE
Log custom plugin stderr only if the status is not ok.

### PR DESCRIPTION
**Pull request by abansal4032**
Otherwise with plugins that run frequently and report ok status, the
logs are filled with unnecessary noise and significantly increases log
size.

### CODE REVIEW

File: pkg/healthchecker/health_checker.go [hunk 1]
```diff
@@ -159,10 +161,9 @@ func execCommand(timeout time.Duration, command string, args ...string) (string,
 	defer cancel()
 
 	cmd := exec.CommandContext(ctx, command, args...)
-	glog.Infof("health-checker: executing command : %v\n", cmd)
 	out, err := cmd.Output()
 	if err != nil {
-		glog.Infof("health-checker: command failed : %v, %v\n", err.Error(), out)
+		glog.Infof("command %v failed: %v, %v\n", cmd, err.Error(), out)
```
Comment (Random-Liu) [inline pkg/healthchecker/health_checker.go:166]: err

File: pkg/healthchecker/health_checker.go [hunk 2]
```diff
@@ -139,14 +141,14 @@ func (hc *healthChecker) CheckHealth() bool {
 	// The service is unhealthy.
 	// Attempt repair based on flag.
 	if hc.enableRepair {
-		glog.Infof("health-checker: component is unhealthy, proceeding to repair")
 		// repair if the service has been up for the cool down period.
 		uptime, err := hc.uptimeFunc()
 		if err != nil {
-			glog.Infof("health-checker: %v\n", err.Error())
+			glog.Infof("error in getting uptime for %v: %v\n", hc.component, err.Error())
```
[the remark this answers is not in the document]
Comment (abansal4032) [reply pkg/healthchecker/health_checker.go]: Done.

File: pkg/healthchecker/health_checker.go (hunk 1 above)
Comment (abansal4032) [reply pkg/healthchecker/health_checker.go]: Done.

**Pull request merged**